### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by tunkukhairiharry**
saje3

### CODE REVIEW

**Review by tunkukhairihermione — COMMENTED** [review]
Could you remove the debug for review?

File: index.html [hunk 1]
```diff
@@ -12,5 +12,7 @@ <h1>
         WELCOME! ENJOY YOUR STAY :DDD
 
     </h1>
+
+    <h2>debug for review</h2>
```
Comment (tunkukhairihermione) [inline index.html]: **THIS**

Comment (tunkukhairihermione) [reply index.html:16]: okay will do

Comment (tunkukhairiharry) [reply index.html]: okay will do

**Review by tunkukhairihermione — CHANGES_REQUESTED** [review]
please do before 10PM

**Review by tunkukhairihermione — APPROVED** [review]
approved

**Pull request merged**